Makefile

```
@@ -0,0 +1,31 @@
+#!/bin/env make
+##
+# Makefile to manage regular operations on this repository
+#
+
+-include devops-tools/Makefile
+
+test:
+	python -m pytest -v tests
+
+coverage:
+	coverage run --branch -m pytest -v tests
+	coverage html -i --omit 'test_*.py,*_test.py,tests/**'
+	coverage report -i --omit 'test_*.py,*_test.py,tests/**'
+
+# Cleans up the generated HTML so that we're not leaving untidy links.
+coverage-cleaned-for-pr:
+	sed -i '/<header/,/<\/header>/d' htmlcov/index.html
+	sed -i '/<aside/,/<\/aside>/d' htmlcov/index.html
+	sed -i '/<head/,/<\/head>/d' htmlcov/index.html
+	sed -i '/<footer>/,/<\/footer>/d' htmlcov/index.html
+	sed -E -i 's/<\/?a[^>]+>//g' htmlcov/index.html
+
+pylint:
+	pylint $(shell git ls-files '*.py')
```

We use ruff for linting now.

```suggestion
pylint:
	ruff check $(shell git ls-files '*.py')
```